model/src/test/kotlin/licenses/ResolvedLicenseInfoTest.kt

```diff
@@ -143,6 +149,177 @@ class ResolvedLicenseInfoTest : WordSpec() {
                 filteredResolvedLicenseInfo.licenses should containLicensesExactly(mit, bsd)
             }
         }
+
+        "filterLicenseResolutions()" should {
+            "remove license that is resolved by a license rule violation resolution" {
+                val resolvedLicenseInfo = ResolvedLicenseInfo(
```

Comment:
The `ResolvedLicenseInfo` object is probably not needed; only the list of licenses is relevant.